services/webhook/general.go

```diff
@@ -28,6 +28,72 @@ func htmlLinkFormatter(url, text string) string {
 	return fmt.Sprintf(`<a href="%s">%s</a>`, html.EscapeString(url), html.EscapeString(text))
 }
 
+// getPullRequestInfo gets the information for a pull request
+func getPullRequestInfo(p *api.PullRequestPayload) (title, link, by, operator, operateResult, assignees string) {
+	if p.PullRequest.HasMerged {
+		p.Action = "merged"
+	}
```

Comment:
It doesn't seem right.

1. The `Action` is defined by `HookIssueAction` and its values. There is no "merged" const for it.
2. This is a unstable behavior, it changes the `p *api.PullRequestPayload`. What if the `p` will be used later? Other code would see a strange `merged` action.
3. Does `p.PullRequest.HasMerged=true` always means `merged`? According to the  `HookIssueAction`  values, it doesn't seem so. Would it cause unrelated actions be sent as `merged`?